doc/user_guide/built_ins.rst

```diff
@@ -135,111 +145,115 @@ the code parser) the differences between the original algorithm code
 and the translated algorithm code are:
 
 * The generic calls to ``invoke`` have been replaced by specific
-  ``CALL invoke_xx``. The calls within the invoke are removed, as are
-  duplicate arguments and any literals leaving the three fields being
-  passed in;
-* A use statement is added for the each of the new ``CALL invoke_xx``
+  ``CALL invoke_<xx>``. The calls within the ``CALL invoke_jacobi_mass_lump``
```

Comment:
Suggest: "... specific calls to ``invoke_<xx>`` routines. The kernel calls within the original ``invoke`` are removed, as are ....literals, leaving the five fields and one operator as arguments;"

Reply:
Thanks for the suggestion. Done.